linetools/scripts/lt_radec.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+"""
+Ingest input coordinates and print them to the screen
+Plot the data for a line (or lines)
+Also print the line data
```

Comment:
What do the above two lines have to do with this script?  Is line data called somehow and I'm missing it?